db/seeds.rb

```diff
@@ -7,3 +7,6 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+def add_norm_seeds_data
+  %w[full_time ,part_time].each { |norm| Norm.create(norm) }
+end
```

Comment:
Who is supposed to call this method? 

Running `rake db:seed` does not populate anything in the db.

![image](https://user-images.githubusercontent.com/2707601/130189686-246581ab-555d-452e-8b9e-3b6333a9c526.png)